requirements/env_hdk.yml

```diff
@@ -18,7 +18,7 @@ dependencies:
   - xlrd>=2.0.1
   - sqlalchemy>=1.4.0,<1.4.46
   - scipy>=1.8.1
-  - matplotlib>=3.6.1
+  - matplotlib
```

Comment:
Why?

Reply:
Conda fails to build the environment due to conflicts.

Reply:
What dependency is causing the conflict?

Reply:
It would be great if we could avoid version inconsistencies between different environments

Reply:
I forgot why do we need this dep at all - for tests?

Reply:
Yes, however, users can use modin on hdk and functions where matplotlib is needed.

Reply:
It’s actually weird that there is a conflict with matplotlib involved - no relations to hdk at all…

Reply:
One of the conflicting packages is openssl. Matplotlib depends on the qt libraries and the last version of qt-main depends on `openssl >=3.1.1,<4.0a0`. PyHDK depends on `folly 2022.11.07.00.*` which depends on `openssl >=1.1.1s,<1.1.2a`.

Reply:
Hmmm… OpenSSL version of pyhdk’s folly looks very old (1.1 vs 3.1/4.0) which is a bit suspicious 
Anyway I think we can try to play with 
Libfolly version in hdk, Probably we could relax current limitations

Reply:
Is it possible to fix the problem fast?

Reply:
I doubt openssl is the reason:

conda search folly=2022.11.07.00 --info -c conda-forge shows

dependencies:
  - boost-cpp >=1.78.0,<1.78.1.0a0
  - bzip2 >=1.0.8,<2.0a0
  - double-conversion >=3.2.0,<3.3.0a0
  - fmt >=8.0.1,<9.0a0
  - gflags >=2.2.2,<2.3.0a0
  - glog >=0.6.0,<0.7.0a0
  - jemalloc
  - libaio >=0.3.113,<0.4.0a0
  - libevent >=2.1.10,<2.1.11.0a0
  - libgcc-ng >=12
  - libjemalloc >=5.2.1
  - libsodium >=1.0.18,<1.0.19.0a0
  - libstdcxx-ng >=12
  - libzlib >=1.2.13,<1.3.0a0
  - lz4-c >=1.9.3,<1.10.0a0
  - openssl >=3.0.7,<4.0a0

Reply:
I can build HDK with the latest version of folly.

Reply:
Do you use mamba or pure conda?

Reply:
I use micromamba:

```
       │  ├─ folly 2022.11.07.00.* , which requires
       │  │  ├─ libevent >=2.1.10,<2.1.11.0a0  but there are no viable options
       │  │  │  ├─ libevent 2.1.10 conflicts with any installable versions previously reported;
       │  │  │  ├─ libevent 2.1.10 would require
       │  │  │  │  └─ openssl >=1.1.1a,<1.1.2a , which can be installed;
       │  │  │  ├─ libevent 2.1.10 would require
       │  │  │  │  └─ openssl >=1.1.1l,<1.1.2a , which can be installed;
       │  │  │  ├─ libevent 2.1.10 would require
       │  │  │  │  └─ openssl >=1.1.1g,<1.1.2a , which can be installed;
       │  │  │  └─ libevent 2.1.10 would require
       │  │  │     └─ openssl >=1.1.1h,<1.1.2a , which can be installed;
       │  │  └─ openssl >=1.1.1s,<1.1.2a , which can be installed;

```

Reply:
Seems openssl restrictions appear because of libevent requirements, I created new build of 0.8 with less restricted libfolly, let's give it a try once it appears in conda-forge channel

Reply:
Build number 1 is in the conda-forge channel now

Reply:
It works in my local env. Updated this PR.